class.jetpack.php

```diff
@@ -2817,9 +2817,19 @@ public static function get_stat_data( $encode = true ) {
 	 * Get additional stat data to sync to WPCOM
 	 */
 	public static function get_additional_stat_data( $prefix = '' ) {
+		global $wpdb;
+
+		$number_of_users = $wpdb->get_var( "SELECT COUNT(*) FROM $wpdb->users" );
```

Comment:
`$number_of_users` is a count of all users on the site. `count_users()` returns an array of different types. `array( admins => 10, editors => 20 etc... );` This can be quite expensive to calculate on really big sites.